src/components/Header/Header.jsx

```diff
@@ -1,5 +1,10 @@
-// import a css file containig header styles
+import React from 'react';
+import './Header.css';
 
-// Create a Header function returning the HTML of header block
+const Header = () => (
+  <header className="header">
+    <h1 className="header__title">Site Name</h1>
```

Comment:
```suggestion
    <h1 className="header__title">Site Name</h1>
    <Navigation />
```
instead App.jsx Navigation component should be rendered here